MIGRATION.md

```diff
@@ -3,3 +3,9 @@ Migration notes
 
 - `project/Build.scala` style based on `sbt.Build` is removed. Migrate to `build.sbt`.
 - `Project(...)` constructor is limited to just two parameters.
+
+- change import in auto plugin to:
+
+```scala
+import sbt._, syntax._, Keys._
```

Comment:
Should we advertise the existence of `sbt.Import`? That's what we use to keep the source compatibility of build definitions, but I guess we don't want to keep it around.